.gitignore

```diff
@@ -0,0 +1,36 @@
+*.class
```

Comment:
Please add checkstyle and rat plugins.
Also suppressions.xml and checkstyle.xml files.

Reply:
Done.